types/webpack/index.d.ts

```diff
@@ -350,7 +351,7 @@ declare namespace webpack {
     type ExternalsElement = string | RegExp | ExternalsObjectElement | ExternalsFunctionElement;
 
     interface ExternalsObjectElement {
-        [key: string]: boolean | string;
+        [key: string]: boolean | string | object;
```

Comment:
this `object` is specifically a `Record<string, string>`, though.

Reply:
@rwaskiewicz could you make the amend suggested please?

Reply:
[In one of the tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b23d51e8daedc06fd41b88304b56577f0f396ed2/types/webpack/webpack-tests.ts#L104) it's mapped to `string[]` or am I missing something?

Reply:
Hmmm. Don't know.